metric_learn/covariance.py

```diff
@@ -14,6 +14,7 @@
 
 from .base_metric import MahalanobisMixin
 from ._util import transformer_from_metric
+import scipy
```

Comment:
Style nitpick: move this import up to line 13 (after the numpy import but before the sklearn)

Reply:
That's right, done